src/css/cs/items/_kitchensink.scss

```diff
@@ -7,6 +7,11 @@
     line-height: normal;
   }
 
+  .uc-page {
+    max-width: 1000px;
+    margin: 0 auto;
```

Comment:
@marcpechaitis  looks like `scss-lint` is failing, just putting `max-width` after `margin` should fix that. Could you push up a fix?